Update index.md Bas document changes (#606)
HL7/fhircast-docs: input/pagecontent/2-4-Subscribing.md

If OAuth 2.0 authentication is used, this POST request SHALL contain the Bearer access token in the HTTP Authorization header.

Hubs SHALL allow subscribers to re-request subscriptions that are already activated. Each subsequent and verified request to a Hub to subscribe or unsubscribe SHALL override the previous subscription state for a specific (`hub.topic`, `hub.channel.endpoint` url) combination. For example, a Subscriber MAY modify its subscription by sending a subscription request (`hub.mode=subscribe`) with a different `hub.events` value with the same topic and endpoint url, in which case the Hub SHALL replace the subscription’s previous `hub.events` with the new list of granted events.

Hubs and Subscribers SHALL be case insensitive for event-names.

The application that creates the subscription might not be the same system as the system connecting to the WSS URL (e.g., a federated authorization model could exist between these two systems). However, in FHIRcast, the Hub assumes that the same authorization and access rights apply to both the Subscriber and the system receiving notifications.

> {% include infonote.html text='The spec uses GET vs POST to differentiate between the confirmation/denial of the subscription request and retrieve the current context. While this is not considered "best practice" from a web architecture perspective, it does make implementation of the callback URL simpler. Since the POST body of the content distribution request may be any arbitrary content type and only includes the actual content of the document, using the GET vs POST distinction to switch between handling these two modes makes implementations simpler.' %}

Reviewer: ```suggestion

```

Confirmation/denial of the subscription request is transmitted over the WSS channel, not  HTTP; this whole callout is a remnant from STU 2, and should be removed.

  
### Subscription Denial

> If (and when) a subscription is denied, the Hub SHALL inform the Subscriber. This can occur for a variety of reasons. Both when a subscription is requested, or after a subscription has already been accepted because the Hub no longer supports that subscription (e.g. it has expired).

Reviewer: ```suggestion
If (and when) a subscription is denied, the Hub SHALL inform the Subscriber. A subscription can be denied when it is  requested or even after having been accepted (e.g. the subscription has expired).

```
The extant sentence is explicitly about chronology, it's best to keep it about that. Simply language.
